diff -crB at-3.1.14/at.c at/at.c
*** at-3.1.14/at.c	2013-09-08 14:43:53.000000000 +0200
--- at/at.c	2014-03-18 14:47:29.885106300 +0100
***************
*** 114,119 ****
--- 114,122 ----
  char *no_export[] =
  {
      "TERM", "DISPLAY", "_", "SHELLOPTS", "BASH_VERSINFO", "EUID", "GROUPS", "PPID", "UID"
+ #ifdef __CYGWIN__
+    ,"ProgramFiles(x86)", "CommonProgramFiles(x86)"
+ #endif
  };
  static int send_mail = 0;
  
